canonical-web-and-design/maas-ui
# refactor(legacy): dedupe the navigation scss

## Done

- Re-use the navigation pattern scss for the legacy and ui workspaces.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View legacy and ui routes, check that the hardware menu only shows at a medium window size.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1800.
